### DISCUSSION TITLE
Expand template numerical sort to work with multiple numbers

### DISCUSSION THREAD

**Discussion by NotAFile** [grafana/grafana#38163] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

With https://github.com/grafana/grafana/pull/5129, you can sort template values numerically. However, this only sorts by the first set of digits (`\d+`) in the string. It would be nice to expand this to sort by multiple numbers.

**Why is this needed**:
The specific case I have encountered this in interface names. Grafana sorts these as follows:

Alphabetical sorting:

```
Ethernet10/1
Ethernet10/10
Ethernet10/2
Ethernet2/1
Ethernet2/10
Ethernet2/2
```

Numerical sorting:
```
Ethernet2/1
Ethernet2/10
Ethernet2/2
Ethernet10/1
Ethernet10/10
Ethernet10/2
```

The order I'd desire:
```
Ethernet2/1
Ethernet2/2
Ethernet2/10
Ethernet10/1
Ethernet10/2
Ethernet10/10
```

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
